alacritty/src/window.rs

```diff
@@ -15,6 +15,7 @@ use std::convert::From;
 #[cfg(not(any(target_os = "macos", windows)))]
 use std::ffi::c_void;
 use std::fmt;
+#[cfg(target_os = "linux")]
```

Comment:
Please, use #[cfg(not(any(target_os = "macos", windows)))] instead, since Linux **and** BSD

Reply:
```suggestion
#[cfg(not(any(target_os = "macos", windows)))]
```